packages/core/src/prompts/prompt.ts

```diff
@@ -104,17 +105,20 @@ export default class Prompt {
 
 		this.input.on('keypress', this.onKeypress);
 		setRawMode(this.input, true);
+		this.output.on('resize', this.render);
 
 		this.render();
 
 		return new Promise<string | symbol>((resolve, reject) => {
 			this.once('submit', () => {
 				this.output.write(cursor.show);
+				this.output.off('resize', this.render);
 				setRawMode(this.input, false);
 				resolve(this.value);
 			});
 			this.once('cancel', () => {
 				this.output.write(cursor.show);
```

Comment:
Time to refactor into a clean up function of sorts? (For both promise branches, I mean)